enhancements/installer/openstack-no-fip-install.md

```diff
@@ -0,0 +1,99 @@
+---
+title: openstack-bring-your-own-external-connectivity
+authors:
+  - "@egarcia"
+  - "@adduarte"
+reviewers:
+  - "@mandre"
+  - "@pierre"
+approvers:
+  - "@mandre"
+  - "@pierre"
+creation-date: 2020-5-26
+last-updated: 2020-5-26
+status: implementable
+---
+
+# OpenStack BYO External Connectivity
+
+## Release Signoff Checklist
+
+- [x] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [x] Test plan is defined
+- [x] Graduation criteria
+- [ ] User-facing documentation is created in [openshift/docs]
+
+## Summary
+
+Numerous customers have requested the ability to run the OpenShift on OpenStack IPI installer without using Floating IP addresses.
+In the installer, we use floating IPs as a way to set up exeternal connectivity to the OpenShift cluster. Depending on how the OpenStack cluster
+is set up, the use of floating IPs may be impossible or very inconvenient. In order to support these customers, we intend to support installations
+on these types of OpenStack clusters.
+
+
+### Non-Goals
+
+This enhancement does not intend to provide any guidance or features that go further than allowing customers to deploy withoug Floating IPs. Operations like attaching loadbalancers to the entrypoints, or any day 2 operations are beyond the scope of this enhancement. 
+
+## Proposal
+
+In order to implement this feature fully, the following changes must be made:
+    1. Make `externalNetwork` an optional argument in the install-config
```

Comment:
Make `externalNetwork` an optional argument in the install-config _when_ user sets `machinesSubnet`.

It's important to only make `externalNetwork` optional when using BYON otherwise this will lead to poor user experience where the install times out. Also, there is no identified use case where deploying on installer-provisioned networks mixed with FIP-less would be useful.